CONTRIBUTING.md

```diff
@@ -0,0 +1,242 @@
+# How to contribute to NumaHOP.
+
+## Did you find a bug ?
+
+- Do not open an issue if the bug is a security vulnerability in our dependencies.
```

Comment:
Then what? Maybe let the reader know that there is no need to open an issue because we will handle security vulnerabilities in our dependencies automatically on our own? And maybe also let them know that can report vulnerabilities in numahop itself through a specific non-public process?